aws/cloudformation/components/alarms.yml.erb

```diff
@@ -82,3 +82,13 @@
     Type: 'AWS::SNS::Topic'
     Properties:
       DisplayName: Infra-Urgent
+
+  StudentLearningWorkdayUrgent:
+      Type: 'AWS::SNS::Topic'
+      Properties:
+        DisplayName: StudentLearning-WorkdayUrgent
+  
+  StudentLearningUrgent:
```

Comment:
Should we go ahead and add a SL-LowPriority while we're here?

Reply:
Yep, that matches my understanding, @sanchitmalhotra126! For @cat5inthecradle's context, an OpenAI outage would cause this tool to error at every attempt to interact with it. It feels within our team's control/context to set up a WorkdayUrgent alarm or notification for us to handle, assuming we're okay only catching these during daytime hours (which we think we are in the very-short-term, since we're talking only English users without a marketing push).

I'm planning on bringing the topic to eng leads this week to figure out "what should we do with alarms that referenced a feature owned by a specific team," which the four of us chatted about a bit in our last meeting.

Reply:
There may be settings within PagerDuty to treat subsequent notifications as part of the same incident. I believe there are also ways to auto-resolve.

My quick opinion on SNS Topics & Pagerduty is that not having an "Urgent" tier kind of makes sense as a policy - The only person who should get woken up after hours is the DOTD and if there's something that can go terribly wrong after hours we should be focusing our efforts on preventing it, rather than designing a complex notification system to wake up the right person.